pkg/sdkclient/sideinput/client.go

```diff
@@ -3,11 +3,12 @@ package sideinput
 import (
 	"context"
 	"fmt"
+	"github.com/numaproj/numaflow-go/pkg/info"
+	"github.com/numaproj/numaflow/pkg/shared/util"
```

Comment:
Please configure your IDE to automatically sort imports. e.g. GoLand

<img width="1161" alt="Screenshot 2024-02-01 at 5 16 22 PM" src="https://github.com/numaproj/numaflow/assets/13165977/4e703a0e-d93c-416b-9d8b-e74602c0b68d">